packages/plugins/RSS3/src/SNSAdaptor/components/FeedCard/common.tsx

```diff
@@ -53,3 +58,22 @@ export const formatValue = (value?: { value: string; decimals: number } | null):
     if (!value) return ''
     return formatBalance(value.value, value.decimals, 5)
 }
+
+const LinkifyRender = ({ attributes, content }: IntermediateRepresentation) => {
+    const { classes, cx } = useStyles()
+    return (
+        <Link {...attributes} className={cx(classes.link, attributes.className)}>
+            {content}
+        </Link>
+    )
+}
+
+export const LinkifyOptions: Opts = {
+    target: '_blank',
+    render: LinkifyRender,
+}
+
+export const htmlToPlain = (htmlString?: string) => {
```

Comment:
Use it in `LinkifyRededer` according to a condition field.

Reply:
We don't use `LinkifyOptions['render']` directly, we just pass it to `Linkify`, and it's been used internally.